datajunction-server/datajunction_server/service_clients.py

```diff
@@ -123,7 +123,7 @@ def submit_query(  # pylint: disable=too-many-arguments
 
     def get_query(
         self,
-        query_id: str,
+        query_id: UUID,
```

Comment:
Yeah, agreed that we probably shouldn't use UUID here. It was set up this way previously but for our internal query service to work with this, we had to do some additional string parsing to get a uuid extracted for this.

Reply:
Yes, taking it all out.